tests/models/test_models.py

```diff
@@ -696,7 +726,7 @@ def test_multiple_covariates(save_path):
     adata.obs["cont2"] = np.random.normal(size=(adata.shape[0],))
     adata.obs["cat1"] = np.random.randint(0, 5, size=(adata.shape[0],))
     adata.obs["cat2"] = np.random.randint(0, 5, size=(adata.shape[0],))
-    setup_anndata(
+    _setup_anndata(
```

Comment:
for this one I guess you would need to set it up once with every method, which feels bad. Although I guess we do not expect this scenario to happen often.